docs/assets/templates/tools/tools.rst

```diff
@@ -0,0 +1,48 @@
+.dockstore.yml for Tools Templates (version 1.2)
+================================================
+Several templates and examples are provided here for you to reference for your own .dockstore.yml files. The last example provides a complete explanation of the possible fields and values you can use.
+
+.. The globalAligner/localAligner example does not seem to correspond to any entry on Dockstore. It's been on github-apps.rst since 2020 (commit 2ee9fb2b10d7b68fbf1481d066651cb8dad855ae).
+
+Simple generic template for a tool
+----------------------------------
+.. include:: /assets/templates/tools/template-minimum.dockstore.yml
+  :code:
+
+Filled-out example of a single tool without a name
```

Comment:
This is good, but it doesn't really say why someone would use a workflow without a name, so someone first coming across this may pick a name arbitrarily just to be complete. It may be worth noting that the main purpose of a name is to distinguish between multiple workflows in one repo.

Reply:
Is there any disadvantage to them arbitrarily choosing a name, unless they are migrating an existing tool/workflow?